packages/@react-spectrum/table/stories/Table.stories.tsx

```diff
@@ -546,6 +548,121 @@ storiesOf('Table', module)
     'async loading',
     () => <AsyncLoadingExample />,
     {chromatic: {disable: true}}
+  )
+  .add(
+    'headerless columns',
+    () => (
+      <Table
+        aria-label="Table with static contents"
+        width={300}
```

Comment:
if we set this to 350, then it'll fit better, right now it's just squeezed enough that both my scrollbars appear